### ISSUE TITLE
Tkinter GUI download progress

### ISSUE THREAD

**Issue by ysc3839** [HelloZeroNet/ZeroBundle#9]
I think it's good. But the ZeroBundle doesn't include tkinter.
https://gist.github.com/ysc3839/208c0db6a40677e54027d339e96ad713

**Comment by HelloZeroNet** [comment]
Yeah, thanks for the code it would be nice to have a progress bar, but i'm not a big fan of tkinter (looks bad, slow startup) and including it to zerobundle would significantly increase the size of it.

So I think the possible solutions:
 - Start packing ZeroNet with ZeroBundle, so it does not needs to download from other sources.
 - Use ctypes based gui library. (pure-python, ligtweight, but win-only)

**Comment by ysc3839** [comment]
I'm good at Win32 development and I can make one using ctypes.
As for Linux, we can use GTK+ with ctypes. But I know nothing about it.
For macOS, it seems that we can't use ctypes to call objc APIs. We have to use pyobjc.

**Comment by HelloZeroNet** [comment]
The third option would be not doing this in python, but using a standard installer, placing desktop/startmenu icon like most user expect. 
Like .msi or nullsoft install system

**Comment by ysc3839** [comment]
I think a "green" software (not require install, not write files into other folders) is good.

**Comment by ysc3839** [comment]
I considered using py2exe in Windows. But I think it's not good to create a data directory in exe's path because this will mess it up. I like macOS's bundle.
